var/spack/repos/builtin/packages/jsoncpp/package.py

```diff
@@ -26,4 +26,4 @@ class Jsoncpp(CMakePackage):
     depends_on('python', type='test')
 
     def cmake_args(self):
-        return ['-DBUILD_SHARED_LIBS=ON']
+        return ['-DBUILD_SHARED_LIBS=ON', '-DJSONCPP_WITH_TESTS=OFF']
```

Comment:
@mdorier can you condition this on whether `self.run_tests` is `True`? by default it will be off. Otherwise I think the result may be unexpected given the python test dependency.

Reply:
Done.